src/screens/Home/InititalAppController.tsx

```diff
@@ -132,14 +137,47 @@ function InititalAppController({ navigation, electrumErrorVisible, setElectrumEr
                     signerMatched,
                     sendAmount,
                     feeRate,
-                  } = generateDataFromPSBT(psbt, signer);
+                  } = generateDataFromPSBT(serializedPSBT, signer);
                   const tnxDetails = getTnxDetailsPSBT(averageTxFees, feeRate);
 
                   if (!signerMatched) {
                     showToast(`Invalid signer selection. Please try again!`, <ToastErrorIcon />);
                     navigation.goBack();
                     return;
                   }
+
+                  if (SignersReqVault.includes(signer.type)) {
```

Comment:
This code block is copy pasted from the other place it's used right? Can we export it to a common function?

Reply:
updated